Formula/s/stp.rb

```diff
@@ -40,12 +40,12 @@ class Stp < Formula
   depends_on "cryptominisat"
   depends_on "gmp"
   depends_on "minisat"
-  depends_on "python@3.12"
+  depends_on "python@3.13"
```

Comment:
I know we didn't have one previously, but would be good to add a minimal test for the Python bindings.

Reply:
Good idea, I added a test and there's something wrong going on with the Python bindings. It's failing on loading `library_path` which is in the installed in the `stp` `site-packages` folder. It seems that cannot be found by the `python3.13` executable.

Reply:
Importing `stp` from within the `site-packages` directory works so it seems to be a matter of getting the `PYTHON_PATH` correctly.